content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,90 @@
+# Continuous Integration and Deployment for Machine Learning
+
+## DevOps
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers), testing, and deployment to production (CI/CD --
+continuous integration and continuous delivery), as well as monitoring &
+feedback.
+
+![](https://static.iterative.ai/img/ml-vs-cicd.png) _Traditional machine
+learning (ML) meets CI/CD_
+
+The main benefits of CI/CD for ML are detailed below.
+
+### Quality Assurance (QA)
+
+This concerns automating & enforcing testing. For example:
+
+- Enforcing data integrity with application-specific tests (such as validation
+  against a schema or verifying pipeline consistency).
+- Enforcing model integrity with application-specific tests (such as
+  input/output and performance validation).
+- When data or code is added/changed in a pull request (PR), automatically
+  running validation tests in the cloud.
+
+### Continuous Machine Learning (CML)
+
+This concerns training in the cloud. For example:
+
+- Infrastructure orchestration: make CI systems provision and launch a GPU
+  instance, train a model, cleanly terminate, and pull results back.
+- Automatically generate performance metrics reports and post them as a PR
+  comment with interactive graphs and tables.
+- Fine-tune on a schedule: set up jobs to pull in new data from regularly
+  updated external sources to refine deployed models.
+- Run a hyperparameter search: keep heavyweight resources and big data on CI
+  servers running overnight, and put your laptop to sleep.
+
+## DVC and CML
+
+<img src="https://static.iterative.ai/logo/dvc.svg" alt="DVC" width="24px" style="vertical-align: text-top"/> +
+<img src="https://static.iterative.ai/logo/cml.svg" alt="CML" width="24px" style="vertical-align: text-top"/>
+= ❤️
+
+Normally, CI/CD is hard to set up, configure, and maintain -- especially for
+data and ML pipelines. DVC can alleviate most (in not all) of the
```

Comment:
Why only mention DVC (and not CML) here?

Reply:
```suggestion
data and ML pipelines. DVC adn CML can alleviate most (in not all) of the
```